openpype/hosts/maya/api/lib.py

```diff
@@ -2111,6 +2111,34 @@ def reset_frame_range():
     cmds.setAttr("defaultRenderGlobals.startFrame", frame_start)
     cmds.setAttr("defaultRenderGlobals.endFrame", frame_end)
 
+    instances = cmds.ls(
+        "*.id",
+        long=True,
+        type="objectSet",
+        recursive=True,
+        objectsOnly=True
+    )
+    frames_attributes = {
+        'frameStart': frame_start,
+        'frameEnd': frame_end,
+        'handleStart': handle_start,
+        'handleEnd': handle_end
+    }
+
+    for instance in instances:
+        if not cmds.attributeQuery("id", node=instance, exists=True):
+            continue
+
+        id_attr = "{}.id".format(instance)
+        if cmds.getAttr(id_attr) != "pyblish.avalon.instance":
+            continue
+
+        for key, value in frames_attributes.items():
+            cmds.setAttr(
```

Comment:
There will be instance nodes that are not time dependent and don't have the relevant attributes. For those this will error.